sdgym/benchmark.py

```diff
@@ -253,157 +267,139 @@ def _run_on_dask(jobs, verbose):
-    in the leaderboard, the old rows are dropped.
-
     Args:
-        synthesizers (function, class, list, tuple or dict):
-            The synthesizer or synthesizers to evaluate. It can be a single synthesizer
-            (function or method or class), or an iterable of synthesizers, or a dict
-            containing synthesizer names as keys and synthesizers as values. If the input
-            is not a dict, synthesizer names will be extracted from the given object.
-        datasets (list[str]):
-            Names of the datasets to use for the benchmark. Defaults to all the ones available.
-        datasets_path (str):
-            Path to where the datasets can be found. If not given, use the default path.
-        modalities (list[str]):
-            Filter datasets by the given modalities. If not given, filter datasets by the
-            synthesizer modalities.
-        metrics (list[str]):
-            List of metrics to apply.
-        bucket (str):
-            Name of the bucket from which the datasets must be downloaded if not found locally.
-        iterations (int):
-            Number of iterations to perform over each dataset and synthesizer. Defaults to 1.
-        workers (int or str):
-            If ``workers`` is given as an integer value other than 0 or 1, a multiprocessing
-            Pool is used to distribute the computation across the indicated number of workers.
-            If ``workers`` is -1, the number of workers will be automatically determined by
-            the number of GPUs (if available) or the number of CPU cores. If the string ``dask``
-            is given, the computation is distributed using ``dask``. In this case, setting up the
-            ``dask`` cluster and client is expected to be handled outside of this function.
-        cache_dir (str):
-            If a ``cache_dir`` is given, intermediate results are stored in the indicated directory
-            as CSV files as they get computted. This allows inspecting results while the benchmark
-            is still running and also recovering results in case the process does not finish
-            properly. Defaults to ``None``.
+        synthesizers (list[class]):
+            The synthesizer(s) to evaluate. Defaults to ``[GaussianCopulaSynthesizer, FASTMLPreset,
+            CTGANSynthesizer]``. The available options are:
+
+                - ``GaussianCopulaSynthesizer``
+                - ``CTGANSynthesizer``
+                - ``CopulaGANSynthesizer``
+                - ``TVAESynthesizer``
+                - ``FASTMLPreset``
+                - any custom created synthesizer or variant
+
+        sdv_datasets (list[str] or ``None``):
+            Names of the SDV demo datasets to use for the benchmark. Defaults to
+            ``[adult, alarm, census, child, expedia_hotel_logs, insurance, intrusion, news,
+            covtype]``. Use ``None`` to disable using any sdv datasets.
+        additional_datasets_folder (str or ``None``):
```

Comment:
Is the functionality for this not being added yet?

Reply:
Not yet. I'll add it in a following PR